_content/gateways/rak2245/device-firmware-setup.md

```diff
@@ -5,7 +5,7 @@ zindex: 800
 
 # Device Firmware Setup
 
->**Note:** If you are planning to use the newly released Raspberry Pi 4 with the RAK2245, use the following firmware which can be downloaded [here](https://downloads.rakwireless.com/en/LoRa/RAK2245-Pi-HAT/Firmware/Raspberry-Pi-4/)
+>**Note:** If you are planning to use the newly released Raspberry Pi 4 with the RAK2245, use the following firmware which can be downloaded [here](https://downloads.rakwireless.com/LoRa/RAK2245-Pi-HAT/Firmware/RAK2245_Latest_Firmware.zip)
```

Comment:
* The word order is a little strange. How about saying: "use the firmware which can be downloaded [here]"

* How about linking to the Download folder, not the specific file? In the future that will prevent the main issue you are trying to solve here.